storage/innobase/handler/ha_innodb.cc

```diff
@@ -2104,8 +2104,9 @@ convert_error_code_to_mysql(
 		/* Since we rolled back the whole transaction, the
 		cached binlog must be emptied. */
 		innodb_transaction_abort(thd, true, error);
-		return error == DB_DEADLOCK
-			? HA_ERR_LOCK_DEADLOCK : HA_ERR_RECORD_CHANGED;
+		if (error == DB_DEADLOCK)
+			return HA_ERR_LOCK_DEADLOCK;
+		return HA_ERR_RECORD_CHANGED;
```

Comment:
Please avoid non-functional changes like this. There also is a white-space-only change to the file `trx0trx.h`.

Reply:
I guess it was a sort of imprecise revert.